runner.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+export TASKS="$(node ./runner/check-aws-ecs-tasks.js)"
+
+if [[ $TASKS == 0 ]]; then
```

Comment:
Sorry, my bash-fu isn't great.  What is this line checking?  At first I thought it was checking the exit code but then I see that is done below